### PR TITLE
Implement NavBar Design to hide on HomePage

### PR DESCRIPTION
**Pull request by AnkitLakkapragada**
### Summary 

This pull request implements the navbar design such that the top one shows on all pages except the home and the larger, bottom navbar shows on the page home. 

### Test Plan 

image
image

### Breaking Changes  

none

### CODE REVIEW

**Comment by dan-ieljin** [comment]
Looks great! The code reads cleanly with conditional rendering logic. New CSS and component updates are consistent and functional.

**Review by cyrus14 — DISMISSED** [review]
Hey Ankit! Great job eliminating the search bar from the home screen. Nice work using CSS to hide the search bar and making the code easy to understand. The new updates are consistent with our current model and work well. Awesome job!

File: frontend/src/components/utils/NavBar/index.tsx [hunk 1]
```diff
@@ -193,23 +193,37 @@ const NavBar = ({ headersData }: Props): ReactElement => {
...
-        </Grid>
-        <Grid item md={6} className={search}>
-          {auto()}
+    if (searchBar) {
+      return (
+        <Grid container className={toolbar} alignItems="center">
+          <Grid item md={3}>
+            {homeLogo}
+          </Grid>
+          <Grid item md={6} className={search}>
```
Comment (thuypham03) [inline frontend/src/components/utils/NavBar/index.tsx:203]: You should prevent duplicate code. For this part, you can combine these 2 into 1 then use `<Grid item md={6} className={searchBar ? search : searchHidden}>` which is the only different line.

File: frontend/src/components/utils/NavBar/index.tsx [hunk 2]
```diff
@@ -193,23 +193,37 @@ const NavBar = ({ headersData }: Props): ReactElement => {
       </Grid>
     </Grid>
   );
-
```
Comment (thuypham03) [inline frontend/src/components/utils/NavBar/index.tsx:196]: Can you help revert those formatting changes (same with other)? Keeping line in middle enhances the readability of our code base.

File: frontend/src/components/utils/NavBar/index.tsx (hunk 1 above)
Comment (AnkitLakkapragada) [reply frontend/src/components/utils/NavBar/index.tsx]: fixed!

**Review by thuypham03 — APPROVED** [review]
Looking good! Congrats on completing your first feature on CUApts team <3 One little thing you can improve on next time is to try reusing existed code and avoiding duplicate as much as possible by passing additional arguments. Again, great job!

**Pull request merged**